.evergreen/compile-windows.sh

```diff
@@ -103,11 +103,13 @@ esac
 if [ "$RELEASE" ]; then
    BUILD_CONFIG="RelWithDebInfo"
    TEST_PATH="./src/libmongoc/RelWithDebInfo/test-libmongoc.exe"
+   FAILING_FLAKY_TEST_OPTS="--skip-failing ../.evergreen/failing-tests --skip-flaky ../.evergreen/flaky-tests"
    export PATH=$PATH:`pwd`/src/libbson/RelWithDebInfo:`pwd`/src/libmongoc/RelWithDebInfo:`pwd`/install-dir/bin
 else
    CONFIGURE_FLAGS="${CONFIGURE_FLAGS} -DENABLE_DEBUG_ASSERTIONS=ON"
    BUILD_CONFIG="Debug"
    TEST_PATH="./src/libmongoc/Debug/test-libmongoc.exe"
+   FAILING_FLAKY_TEST_OPTS="--skip-failing .evergreen/failing-tests --skip-flaky .evergreen/flaky-tests"
```

Comment:
The path is correct.  If you look at L31-36, you'll see that when `$RELEASE` is set (and thus we are building from the release tarball), the process will descend into a sub-directory where the actual build takes place.  However, since these failing/flaky test skips are meant for Evergreen, we don't ship the files in the release tarball, so they have to be referenced relative to the parent directory.

Reply:
Ah that makes sense. I missed L31-36. Thank you for the explanation.